src/styles/main.scss

```diff
@@ -1,3 +1 @@
-body {
-  margin: 0;
-}
+@import "header", "navigation", "container", "card", "reset";
```

Comment:
You must specify the path to the file to import it.
For example, `@import './header;'
And you need to import one file at a time.